### DISCUSSION TITLE
%essentials_safe_online% showing blank value

### DISCUSSION THREAD

**Discussion by sandornot** [PlaceholderAPI/PlaceholderAPI#964] [(2) Support] [answered]
![702E61AF-A1B4-4325-9AAA-CEDAE2845DF6](https://github.com/PlaceholderAPI/PlaceholderAPI/assets/119451522/aa6eff7e-2ef2-40a3-aa14-800c1acced44)
I'm trying to use the %essentials_safe_online% placeholder in my DiscordSRV bot's status but instead of it saying like "8 players" it only says " players", does anyone know what the issue could be

**Comment by iGabyTM** [comment, marked as answer]
Open an issue on https://github.com/PlaceholderAPI/Essentials-Expansion/issues if this is still a problem and the placeholder doesn't work at all - not only on DSRV.
